plugins/inputs/slurm/README.md

```diff
@@ -0,0 +1,181 @@
+slurm_jobs,host=hoth,job_id=16965,name=gridjob,source=slurm_primary.example.net command="/tmp/SLURM_job_script.OHQ65g",cpus=2i,current_working_directory="/home/sessiondir/9tIMDmS6at5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmkaIKDmU2Bmin",group_id=2005i,nice=50i,node_count=1i,nodes="naboo228",partition="atlas",priority=4294884764i,standard_error="/home/sessiondir/9tIMDmS6at5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmkaIKDmU2Bmin.comment",standard_input="/dev/null",standard_output="/home/sessiondir/9tIMDmS6at5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmkaIKDmU2Bmin.comment",start_time=1722520367i,state="COMPLETED",state_reason="None",submit_time=1722520366i,tasks=1i,time_limit=3600i,tres_req_str="cpu=1,mem=1000M,node=1,billing=1" 1722599914000000000
+slurm_jobs,host=hoth,job_id=17036,name=gridjob,source=slurm_primary.example.net command="/tmp/SLURM_job_script.DmbM7i",cpus=8i,current_working_directory="/home/sessiondir/M59LDmmyct5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmzbIKDmEzwdwn",group_id=2005i,nice=50i,node_count=1i,nodes="naboo225",partition="atlas",priority=4294884693i,standard_error="/home/sessiondir/M59LDmmyct5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmzbIKDmEzwdwn.comment",standard_input="/dev/null",standard_output="/home/sessiondir/M59LDmmyct5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmzbIKDmEzwdwn.comment",start_time=1722543684i,state="RUNNING",state_reason="None",submit_time=1722543684i,tasks=8i,time_limit=3600i,tres_req_str="cpu=8,mem=8000M,node=1,billing=8" 1722599914000000000
+slurm_jobs,host=hoth,job_id=17483,name=gridjob,source=slurm_primary.example.net command="/tmp/SLURM_job_script.UM7Uy8",cpus=1i,current_working_directory="/home/sessiondir/PLuMDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmGiIKDmy0UFLn",group_id=2005i,nice=50i,node_count=1i,nodes="naboo145",partition="atlas",priority=4294884246i,standard_error="/home/sessiondir/PLuMDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmGiIKDmy0UFLn.comment",standard_input="/dev/null",standard_output="/home/sessiondir/PLuMDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmGiIKDmy0UFLn.comment",start_time=1722597933i,state="RUNNING",state_reason="None",submit_time=1722597932i,tasks=1i,time_limit=3600i,tres_req_str="cpu=1,mem=2000M,node=1,billing=1" 1722599914000000000
+slurm_jobs,host=hoth,job_id=17484,name=gridjob,source=slurm_primary.example.net command="/tmp/SLURM_job_script.c7w3hs",cpus=1i,current_working_directory="/home/sessiondir/Y4kNDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmHiIKDmd9hu4m",group_id=2005i,nice=50i,node_count=1i,nodes="naboo145",partition="atlas",priority=4294884245i,standard_error="/home/sessiondir/Y4kNDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmHiIKDmd9hu4m.comment",standard_input="/dev/null",standard_output="/home/sessiondir/Y4kNDmUV2t5nKG01gq4B3BRpm7wtQmABFKDmbnHPDmHiIKDmd9hu4m.comment",start_time=1722598133i,state="RUNNING",state_reason="None",submit_time=1722598133i,tasks=1i,time_limit=3600i,tres_req_str="cpu=1,mem=2000M,node=1,billing=1" 1722599914000000000
```

Comment:
This looks like something that needs additional parsing? `tres_req_str="cpu=1,mem=2000M,node=1,billing=1"`

Reply:
I thought about parsing that, but didn't want to get in over my head in terms of resilience. I have added some parsing logic and the tests are passing 2e3a0ffd9df58a1545014c2c229e5b4946090b8b.

Please do let me know about your thoughts for this one...

Thanks for pointing it out!

Reply:
Looks good to me 👍